recipes-openxt/qemu-dm/qemu-dm-2.6.2/exit-mainloop-on-reset.patch

```diff
@@ -0,0 +1,61 @@
+################################################################################
```

Comment:
I get what this is doing and it is fine. Maybe we can improve things in the future since QEMU handles restarts and power state changes much better these days. Maybe an improvement ticket could be created to investigate that.